src/java/main/org/apache/zookeeper/server/quorum/QuorumPeerConfig.java

```diff
@@ -292,6 +293,8 @@ public void parseProperties(Properties zkProp)
                 }
             } else if ((key.startsWith("server.") || key.startsWith("group") || key.startsWith("weight")) && zkProp.containsKey("dynamicConfigFile")) {
                 throw new ConfigException("parameter: " + key + " must be in a separate dynamic config file");
+            } else if(key.equals("warn.responseTime")) {
```

Comment:
I will document the new property and change the name to the suggested one.
I have tested this feature manually. 
I am currently trying to add unit test case in NettyServerCnxnTest.java or NIOServerCnxnTest.java, but seems too high level approach.
Can you suggest any locations for adding test case? @hanm